### PR TITLE
Change to use RESTEasy Reactive.

### PR DESCRIPTION
**Pull request by phillip-kruger**
All admin related BD operation as still being performed blocking. This is to change the GET `/maven` from the eventloop

Also added a banner (for fun).

Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -55,11 +55,19 @@
     </dependency>
     <dependency>
       <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-resteasy-jackson</artifactId>
+      <artifactId>quarkus-resteasy-reactive</artifactId>
```
Comment (gastaldi) [inline pom.xml:58]: This is resolved transitively through `quarkus-resteasy-reactive-jackson`

File: pom.xml [hunk 2]
```diff
@@ -55,11 +55,19 @@
...
-      <artifactId>quarkus-resteasy-qute</artifactId>
+      <artifactId>quarkus-resteasy-reactive-jackson</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy-reactive-qute</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-hibernate-validator</artifactId>
```
Comment (gastaldi) [inline pom.xml:70]: Is this extension being used anywhere?

**Review by gastaldi — COMMENTED** [review]
That looks okay, but I was expecting to go full reactive (returning `Uni`s) instead of having blocking operations (which in theory won't change a thing).

File: pom.xml (hunk 2 above)
Comment (phillip-kruger) [reply pom.xml:70]: I got an error that needed that, Iet me double check

File: pom.xml (hunk 1 above)
Comment (phillip-kruger) [reply pom.xml:58]: Cool, I'll remove

**Comment by phillip-kruger** [comment]
For now, because we are caching, and I don't think we can cache Uni, this change really just change the thread it's executing on

File: pom.xml (hunk 2 above)
Comment (phillip-kruger) [reply pom.xml:70]: Yes, Admin API use it. Keeping it here for now. I assume it was pulled in transitively before, and not anymore

**Review by gastaldi — APPROVED** [review]

**Comment by gastaldi** [comment]
I remember the reason why I didn't use Hibernate Reactive was because the Reactive PostgreSQL driver didn't support columns with the JSON data type, need to check if that was fixed (or if I even reported it :)).

**Comment by phillip-kruger** [comment]
Ok, cool. I still could not find panache reactive, but like I said, for what we want to do this should be ok.

**Pull request merged**